ajax/libs/androidicons/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "androidicons",
+  "version": "1.0",
+  "description": "Androidicons is a professional, handcrafted icon set with 250 icons in 14 different colors and 5 sizes, including an icon font and all sources",
+  "filename": "css/androidicons.min.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/opoloo/androidicons.git"
+  },
+  "author": "Jochen Greif",
+  "homepage": "http://www.androidicons.com/",
+  "keywords": [
+    "android",
+    "icons"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/opoloo/androidicons.git",
+    "fileMap": [
+      {
+        "basePath": "iconfont",
+        "files": [
+          "css/*",
+          "font/*"
```

Comment:
Let's use `+(css|font)/*`
As usual, we'd like to reduce the line of the pattern, and to get all the files we want.
Thanks.